src/main/java/org/jboss/pnc/api/deliverablesanalyzer/dto/Build.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2020 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.api.deliverablesanalyzer.dto;
+
+import java.util.Objects;
+import java.util.Set;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.PositiveOrZero;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.ToString;
+import lombok.extern.jackson.Jacksonized;
+
+/**
+ *
+ * @author jbrazdil
+ */
+@Data
+@Builder(builderClassName = "Builder")
+@AllArgsConstructor
+@ToString
+@Jacksonized
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Build {
+
+    private final BuildSystemType buildSystemType;
+
+    @PositiveOrZero
+    private final Long kojiId;
+
+    @NotEmpty
+    private final String kojiNVR;
+
+    @NotEmpty
+    private final String pncId;
```

Comment:
What should be its content?
I think it can be null in case of a Brew build.

Reply:
in case of Brew build it can be null, otherwise it is non-empty string.

Reply:
Then I think this should be changed as NotEmpty doesn't allow null value.